packages/client/src/components/Post.js

```diff
@@ -138,6 +139,8 @@ const Post = ({ title, body, isDraft, isAdmin, id }) => {
                   Edit
                 </Button>
               </ButtonDiv>
+            ) : (
+              <> </>
```

Comment:
This can just return `null` instead so react doesn't attempt to render.
```suggestion
              null
```